tests/ci/data/profiles/rosa-hcp.yaml

```diff
@@ -151,3 +151,26 @@ profiles:
   account-role:
     path: ""
     permission_boundary: ""
+- as: rosa-hcp-arm # id:75108
```

Comment:
Except for the ARM instance, there is not much difference from the previous profiles. In my opinion, the better way for us is to open the Global variable 'INSTANCE_TYPE' and build ARM jobs with the existing profiles rather than add a new one.

Reply:
Removed the profile. Added the `COMPUTE_MACHINE_TYPE` env configuration